### DISCUSSION TITLE
About launching DEX exchage on GOchain for my own

### DISCUSSION THREAD

**Discussion by PEcoinCEO** [gochain-io/docs#133] [Q&A]
I made a coin based on GO chain server.
I want to create a DEX exchange so that my coin can be traded.
I found something called swithdex as shown in the figure below,

https://switchdex.ag/#/erc20/dex-wizard?quoteToken=weth&token=ghost

 This is a site that launchs the dex exchange with one-click and it is based on Ethereum. I tried to make it, but I failed. Can you help me make it by putting the coding on top of the GO server? It seems that the coding information is Jason(configFile(2).Jason) of the above site. Can you put it in as it is and make it in a GO chain server? 

Adapting the coding information above, could you make dex exchange with one-click for me as you did last time for me? (like making coin with one-click)

I wish you all the best. I wish I could get information that would help.

**Comment by treeder** [comment]
No, we can't make you a DEX, but feel free to launch your own or take their code and point it at the right spot (GoChain RPC). 

If you just want to use a dex, there's already https://goswap.exchange.

**Comment by PEcoinCEO** [comment]
Thank you.treeder
